pkg/operator/ceph/cluster/controller.go

```diff
@@ -118,11 +122,14 @@ func newReconciler(mgr manager.Manager, ctx *clusterd.Context, clusterController
 		panic(err)
 	}
 
+	clusterController.recorder = mgr.GetEventRecorderFor("ClusterController")
+
 	return &ReconcileCephCluster{
 		client:            mgr.GetClient(),
 		scheme:            mgrScheme,
 		context:           ctx,
 		clusterController: clusterController,
+		recorder:          mgr.GetEventRecorderFor("ClusterController"),
```

Comment:
You can probably leave this out and have a pointer to the recorder only in the clusterController.

Reply:
This can be done very easily.